api/src/main/java/io/smallrye/reactive/messaging/MessagePublisherProvider.java

```diff
@@ -0,0 +1,14 @@
+package io.smallrye.reactive.messaging;
+
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.reactivestreams.Publisher;
+
+/**
+ * Framework-facing interface for the Emitter implementations.
+ *
+ * @param <T> message payload type.
```

Comment:
Minor: maybe mention that the emitter is emitting Message<T>. While the current wording is correct, it led to confusion.

Reply:
Done